cmd/up/activate.go

```diff
@@ -180,16 +178,25 @@ func (up *upContext) activate(autoDeploy, build bool) error {
 		printDisplayContext(up.Dev, divertURL)
 		if hook == "yes" {
 			log.Information("Running start.sh hook...")
+			if !up.isRetry {
```

Comment:
Let's remove this one, is part of the `up` sequence

Reply:
If we remove this one, we will send the event every time it execute the command on the dev pod

Reply:
What I mean is to remove the full `if` block and just keep https://github.com/okteto/okteto/pull/1654/files#diff-c39b965c9f4376229cb1d945d1dcddaca021d99664fe2c5674aee1c278999600R190

Reply:
Ah, I move setting  `isRetry` code down because it was sending retry events in the first iteration of the command  https://github.com/okteto/okteto/blob/master/cmd/up/activate.go#L138